### ISSUE TITLE
Manage cookies

### ISSUE THREAD

**Issue by pcejas** [pichillilorenzo/flutter_inappbrowser#8]
Please add a way to manage cookies from Dart (set, get) for both platforms.

**Comment by pichillilorenzo** [comment]
Yeah, I will add this, but for iOS will be available only for **iOS 11.0+**: [WKHTTPCookieStore](https://developer.apple.com/documentation/webkit/wkhttpcookiestore?changes=latest_minor)
